tasks/xgettext.js

```diff
@@ -228,14 +242,15 @@ module.exports = function(grunt) {
         extractDirectiveStrings('"', func);
       });
 
-      return messages;
+      return [messages, messageToFilesMap];
     },
 
     handlebars: function(file, options) {
       var contents = grunt.file.read(file).replace("\n", " "),
         fn = _.flatten([ options.functionName ]),
```

Comment:
Flatten can be achieved better natively with reduce and rest arguments.
[options.functionName].reduce((a, b) => [ ...a, ...b ]);